src/helix/endpoints/chat/send_chat_message.rs

```diff
@@ -0,0 +1,266 @@
+    /// If Twitch recognizes the name, Twitch converts the name to the emote
+    /// before writing the chat message to the chat room.
+    #[cfg_attr(feature = "typed-builder", builder(setter(into)))]
+    #[cfg_attr(feature = "deser_borrow", serde(borrow = "'a"))]
+    pub message: Cow<'a, str>,
+    /// The ID of the chat message being replied to.
+    #[cfg_attr(feature = "typed-builder", builder(setter(into)))]
+    #[cfg_attr(feature = "deser_borrow", serde(borrow = "'a"))]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub reply_parent_message_id: Option<Cow<'a, types::MsgIdRef>>,
+}
+
+impl<'a> SendChatMessageBody<'a> {
+    /// Send a message in the broadcaster's channel
+    pub fn new(
+        broadcaster_id: impl types::IntoCow<'a, types::UserIdRef> + 'a,
+        sender_id: impl types::IntoCow<'a, types::UserIdRef> + 'a,
+        message: impl types::IntoCow<'a, str> + 'a,
+    ) -> Self {
+        Self {
+            broadcaster_id: broadcaster_id.into_cow(),
+            sender_id: sender_id.into_cow(),
+            message: message.into_cow(),
+            reply_parent_message_id: None,
+        }
+    }
+
+    /// Set the reply parent message-id
+    pub fn reply_parent_message_id(
+        mut self,
+        reply_parent_message_id: impl types::IntoCow<'a, types::MsgIdRef> + 'a,
+    ) -> Self {
+        self.reply_parent_message_id = Some(reply_parent_message_id.into_cow());
+        self
+    }
+}
+
+impl helix::private::SealedSerialize for SendChatMessageBody<'_> {}
+
+impl<'a> helix::HelixRequestBody for [SendChatMessageBody<'a>] {
+    fn try_to_body(&self) -> Result<hyper::body::Bytes, helix::BodyError> {
+        #[derive(Serialize)]
+        struct InnerBody<'a> {
+            data: &'a [SendChatMessageBody<'a>],
+        }
+
+        serde_json::to_vec(&InnerBody { data: self })
+            .map_err(Into::into)
+            .map(Into::into)
+    }
+}
+
+/// A drop reason of a sent message.
+#[derive(PartialEq, Eq, Deserialize, Serialize, Debug, Clone)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct ChatMessageDropReason {
+    /// Code for why the message was dropped.
+    pub code: String,
+    /// Message for why the message was dropped.
+    pub message: String,
+}
+
+/// Return Values for [Send Chat message](super::send_chat_message)
+///
+/// [`send-chat-message`](https://dev.twitch.tv/docs/api/reference#send-chat-message)
+#[derive(PartialEq, Eq, Deserialize, Serialize, Debug, Clone)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct SendChatMessageResponse {
+    /// The message id for the message that was sent.
+    pub message_id: types::MsgId,
+    /// If the message passed all checks and was sent.
+    pub is_sent: bool,
+    /// The reason the message was dropped, if any.
+    pub drop_reason: Option<ChatMessageDropReason>,
```

Comment:
`msg_id` can be empty

```json
{"data":[{"message_id":"","is_sent":false,"drop_reason":{"code":"msg_rejected","message":"Your message is being checked by mods and has not been sent."}}]}
```


Should be `Option<types::MsgId>` and be annotated with `#[serde(deserialize_with = "helix::deserialize_none_from_empty_string")]`